### PR TITLE
Add iOS build support (OpenGL)

### PR DESCRIPTION
**Pull request by Gordon-F**
TODO:
- [x] Update CI
- [ ] ???

Add iOS build support. Just copy-paste sapp-darwin version with regenerated bindings.

Also you can see [sample project](https://github.com/Gordon-F/miniquad_ios_example)

IMG_0814

### CODE REVIEW

**Comment by not-fl3** [comment]
Wow, this is awesome!

File: README.md [hunk 1]
```diff
@@ -114,7 +115,7 @@ basic-http-server .
 
 ## android
 
-Recommended way to build for android is using Docker.   
+Recommended way to build for android is using Docker.
```
Comment (not-fl3) [inline README.md:118]: Trailing whitespaces in markdown make a newline. 
Without those whitespaces, formatting in README on GH is slightly corrupted :)

Comment (mvlabat) [reply README.md:118]: `<br/>` should work as well, and it doesn't get removed by the tools that clean trailing spaces up automatically

Comment (not-fl3) [reply README.md:118]: >  <br/> should work as well, and it doesn't get removed by the tools that clean trailing spaces up automatically

Good point, <br/> may be easier to maintain.

Comment (Gordon-F) [reply README.md]: @mvlabat thanks! Fix it.

**Comment by not-fl3** [comment]
Merged, crates version updated. 

Thank you very much for this PR, I really like how easy are the build steps of example project. 
A huge step for miniquad! 

🎉🎉🎉

**Pull request merged**